repositories/repositories.bzl

```diff
@@ -219,13 +219,13 @@ def ros2_repositories():
         urls = ["https://github.com/chriskohlhoff/asio/archive/refs/tags/asio-1-27-0.tar.gz"],
     )
 
+    # We're pointing at hedronvision's mirror of google/boringssl:master-with-bazel to get Renovate auto-update. Otherwise, Renovate will keep moving us back to master, which doesn't support Bazel. See https://github.com/renovatebot/renovate/issues/18492
```

Comment:
Please split this very long line into a couple of shorter one. With e.g. 120 chars max per line.

Reply:
I'd remove `See https://github.com/renovatebot/renovate/issues/18492` -- that already stands in hedronvision/boringssl repo.

Reply:
Done